main.go

```diff
@@ -268,3 +276,20 @@ func bashCompleteSnapshotterNames(clicontext *cli.Context) {
 		fmt.Fprintln(clicontext.App.Writer, name)
 	}
 }
+
+func globalFlags(clicontext *cli.Context) (string, []string) {
+	args0 := os.Args[0]
+	if runtime.GOOS == "linux" {
+		var err error
+		args0, err = os.Readlink("/proc/self/exe")
```

Comment:
you can try to use `os.Executable()` here :)  it can be used for windows

Reply:
fixed